### ISSUE TITLE
Movies from Radarr that use Symlinks don't evaluate to the correct path

### ISSUE THREAD

**Issue by ElEmFurt** [morpheus65535/bazarr#2415]
**Describe the bug**
I have a ton of deep nested movies for example - Collections/Movie Collection/Movie (Year)/Movie.mp4 and I wrote a powershell script to flatten the folder structure using symlinks and have all the movie folders in a single folder instead of the nested path, so that Radarr would be able to scan them. While this worked fine and even got imported into Bazarr, Bazarr doesn't evaluate the path properly.

I also had to map my path since the movies are on an external drive, so for e.g. my symlinks folder is **F:\ColSym** and I have mapped it to **\\192.168.0.1\h\Movies\Collections**, this allows for the movies to be scanned but with the wrong path in Bazarr. E.g. a movie with the path `\\192.168.0.1\h\Movies\Collections\Undisputed Series\Undisputed (2002)\` and a symlink of `F:\ColSym\Undisputed (2002)` becomes `\\192.168.0.1\h\Movies\Collections\Undisputed (2002)\` because the series folder wasn't mapped.

My issue is that I have around 260 series\collection folders and I think Bazarr should have properly mapped the folders when scanning them from Radarr, especially if it was a symlink. Is there any way to handle this as I am losing the subtitle feature for a ton of movies.

**Comment by JaiZed** [comment]
With such a massive change, Bazarr might still be looking at the old paths, if they still work.

At a minimum, you should run the "Index All Movie Subtitles from Disk" Task first.

However, please use the Discord support channel first.
This board is for confirmed bug tracking.

**Comment by ElEmFurt** [comment]
I haven't moved the files, they have always been in these paths since the start, only thing is that since Bazarr is dependent on Radarr and Radarr doesn't support deep nested paths, I had to first figure out how to get it working there, which I have, only that Bazarr can't figure out the actual path from the symlinks.

I will use the Discord for future posts, thanks for letting me know.

**Comment by morpheus65535** [comment]
Bazarr doesn't "resolve" symlink, it use it as provided by Radarr and do a simple "search and replace" in the provided path when using path mapping.

**Comment by ElEmFurt** [comment]
Incase anyone is seeking an answer to this (I am still working on an automated solution), I figured out the root of the problem is Radarr, which needs me to explicitly add all my second level (series) folders. I was able to add one of the series folders using the API at [https://radarr.video/docs/api/#/](url) using the rootFolder API call, and passing the folder path in the body of the request.

I am now working on a script to automate running through my folders and adding the missing ones to Radarr so that Bazarr can also get the subtitles automatically.
